test/yorkie_test.ts

```diff
@@ -453,18 +454,18 @@ describe('Yorkie', function () {
     await withTwoClientsAndDocuments(async (c1, d1, c2, d2) => {
       // 01. Updates 700 changes over snapshot threshold.
       for (let idx = 0; idx < 700; idx++) {
-        d1.update((root) => {
+        d1.update((root: Indexable) => {
           root[`${idx}`] = idx;
         });
       }
       await c1.sync();
 
       // 02. Makes local changes then pull a snapshot from the agent.
-      d2.update((root) => {
+      d2.update((root: Indexable) => {
         root['key'] = 'value';
       });
       await c2.sync();
-      assert.equal(d2.getRootObject()['key'], 'value');
+      assert.equal(d2.getRootObject() as Indexable['key'], 'value');
```

Comment:
The problem of breaking the CI is related to the code below. I have confirmed that the problem has been fixed when modifying it as below.
```suggestion
      assert.equal((d2.getRootObject() as Indexable)['key'], 'value');
```